packages/mobile/public/index.html

```diff
@@ -7,9 +7,9 @@
         <title>IOTA desktop app</title>
 
         <link rel="icon" type="image/png" href="/favicon.png" />
-        <link rel="stylesheet" href="/build/bundle.css" />
+        <link rel="stylesheet" href="/build/index.css" />
```

Comment:
Optionaly add types to the link and script tags like in the favicon.
`<link rel="stylesheet" type="text/css" href="/build/index.css" /> `

Reply:
Thanks for your suggestion, but I can't see your point in this case, it seems redundant to me. There are different approaches related to defining defaults, but I'm reluctant to add optional values to any code, since the less we write the less error-prone. IMHO knowing the defaults and using / defining them into our own code is key.

WHATWG specs define the default type for resources given by the stylesheet keyword is text/css.
https://html.spec.whatwg.org/multipage/links.html#link-type-stylesheet

Maybe I'm loosing something, what you think could be gained adding optional type prop in this case?